### PR TITLE
add ncplot package

### PR DESCRIPTION
**Pull request by r4ecology**
Checklist
- [X ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ X] Source is from official source
- [ X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X ] If static libraries are linked in, the license of the static library is packaged.
- [ X] Build number is 0
- [ X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by r4ecology** [comment]
@conda-forge-admin I added this pull request to add a new package , ncplot, a couple of weeks. Let me know if something is wrong with the PR.

File: recipes/ncplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv 
+
+
+requirements:
+
+  host:
+    - python
+    - pip
+    - setuptools_scm
```
Comment (dopplershift) [inline recipes/ncplot/meta.yaml:21]: `setuptools_scm` doesn't seem to be used by the package?

**Comment by dopplershift** [comment]
Other than the minor comment above, this looks ready to go. Do you want to update to the latest version with updated dependencies before merging?

**Comment by r4ecology** [comment]
Thanks @dopplershift. I have updated the recipe so it's pulling in the latest pypi version. I've had to add numpy to host, as cartopy strangely requires numpy to be pre-installed.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**